### DISCUSSION TITLE
Would it be feasible to switch to Goanna, Gecko, Servo, or even Webkit? Blink doesn't follow standards and is killing the open web

### DISCUSSION THREAD

**Discussion by xplshn** [minbrowser/min#2281] [Completed]
That's it. Would it be feasible to switch to Goanna, Gecko, Servo, or even Webkit? Blink doesn't follow standards and is killing the open web?

Since Blink is not a standard's respecting engine. Maybe we could compile it with Thorium patches?

**Comment by PalmerAL** [comment]
Goanna and Servo don't have good website compatibility (Servo in particular is essentially a research project - go try out the builds here: https://servo.org/download/. It's not feasible to use as a daily browser).

Webkit or Gecko would work well, but there isn't a good Electron equivalent for them - we need something that handles embedding them, providing access to native APIs through javascript, and packaging.

Related: #1922

**Reply by xplshn** [reply]
Gecko can be embedded, IDK about Goanna, Servo can, as well

**Reply by PalmerAL** [reply]
Yup, but there isn't a prebuilt system of embedding them and integrating with nodeJS. It's possible, but would be a lot of work.
